mcd/mcd-seth/mcd-seth.md

```diff
@@ -1,20 +1,23 @@
-# Drawing Dai from the Kovan Maker Protocol deployment using Seth
+# Drawing Dai from the Goerli Maker Protocol deployment using Seth
 
-**This guide works under the Kovan [1.0.2 Release](https://changelog.makerdao.com/releases/kovan/1.0.2/index.html) of the system.**
-
-- [Drawing Dai from the Kovan Maker Protocol deployment using Seth](#drawing-dai-from-the-kovan-maker-protocol-deployment-using-seth)
-  - [Prerequisites](#prerequisites)
-    - [Setting up Seth](#setting-up-seth)
-    - [Other tools](#other-tools)
-  - [Getting tokens](#getting-tokens)
-  - [Saving contract addresses](#saving-contract-addresses)
-  - [Token approval](#token-approval)
-  - [Finally interacting with the Maker Protocol contracts](#finally-interacting-with-the-maker-protocol-contracts)
-  - [Paying back DAI debt to release collateral](#paying-back-dai-debt-to-release-collateral)
+**This guide works under the Goerli [1.11.0 Release](https://changelog.makerdao.com/releases/goerli/1.11.0/index.html) of the system.**
 
 **You can also use the Rinkey, Ropsten and Goerli deployments in this guide. Just make sure to change the contract addresses from the specific network.**
 
-This tutorial will cover how to use the tool `seth` to deposit BAT tokens to draw DAI from the Kovan deployment of MCD as an example, since the process is the same for any other ERC-20 token. You can use the same methodology for any supported token in MCD, by changing the contract addresses to the specific token you want to use.
+This tutorial will cover how to use the tool `seth` to deposit WBTC tokens to draw DAI from the Goerli deployment of MCD as an example, since the process is the same for any other ERC-20 token. You can use the same methodology for any supported token in MCD, by changing the contract addresses to the specific token you want to use.
+
+<!-- vim-markdown-toc GFM -->
```

Comment:
It seems that some vim artifacts were introduced alongside the changes. I would suggest to cleanup those.
```suggestion
```